### PR TITLE
Update editor installation instructions based on feedback

### PR DESCRIPTION
**Pull request by olafurpg**
Followup on feedback from several people for things that can be clearer in the docs.

I'm not too happy about the readability of the markdown sources with the `mdoc:generic` modifiers. I can live with it however, since it avoids copy-pasting a lot of text. I want to try and have each editor page be fully self-contained even if it comes at the price of duplicated content.

### CODE REVIEW

File: docs/editors/vim.md [hunk 1]
```diff
@@ -58,6 +45,19 @@ let g:lsc_auto_map = {
     \}
 ```
 
+Run `:PlugInstall` to install the plugin. If you already have `vim-lsc`
```
Comment (gabro) [inline docs/editors/vim.md]: Run PlugInstall (or the relevant command for your plugin manager)

**Comment by olafurpg** [comment]
@gabro can you elaborate?

One thing we could do is support importing markdown from other files.

````
```scala mdoc:inline:generic.md
```
````

That would then read `generic.md` and inline the contents.

**Comment by gabro** [comment]
Yeah that's what I meant. It would at least improve the UX of editing those files.

**Comment by olafurpg** [comment]
The tricky part about inlining is that we may want to process the inlined doc with mdoc, I'm just gonna stop now and keep the current approach since it's working OK-ish

File: docs/editors/vim.md (hunk 1 above)
Comment (olafurpg) [reply docs/editors/vim.md]: It already says "assuming vim-plug above" and highlights vundle works too. Nobody has complained about these instructions so far so I guess it's been fine. The main improvement in this PR was to move all `~/.vimrc` config into one code listing.

**Pull request merged**